aiosqlite/core.py

```diff
@@ -47,13 +47,15 @@ def __init__(
         connector: Callable[[], sqlite3.Connection],
         iter_chunk_size: int,
         loop: Optional[asyncio.AbstractEventLoop] = None,
+        parent_loop: Optional[asyncio.AbstractEventLoop] = None,
```

Comment:
Is there a reason or benefit to explicitly passing the parent loop (something that was explicitly deprecated in previous versions), rather than just using `get_loop` to get the active loop when the connection object is made? Using `get_loop` instead will prevent errors if the non-active loop is passed in, and also provide more consistent behavior in the `run()` method.

I think I would also prefer tracking this as `self._loop` for brevity.

Reply:
I added it as an explicit parameter to preserve existing behavior and because `get_loop` may not immediately run, like if `self._tx` is never filled with any futures, and it would pick the first-seen event loop in the case of multiple loops, which may not be desired.

Reply:
If `get_loop` is called within the `_connect` method, then there's never a chance that it would get the wrong event loop. I'd overall prefer to have consistent behavior with minimal parameters than have more parameters that result in a higher number of distinct codepaths.